packages/braid-design-system/src/lib/components/icons/IconEnlarge/IconEnlarge.docs.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import type { ComponentDocs } from 'site/types';
+import { iconDocumentation } from '../iconCommon.docs';
+import source from '../../../utils/source.macro';
+import {
+  IconEnlarge,
+  Heading,
+  Stack,
+  Strong,
+  Text,
+  Inline,
+  Button,
+} from '../../';
+
+const docs: ComponentDocs = {
+  category: 'Icon',
+  migrationGuide: true,
+  Example: () =>
+    source(
+      <Stack space="none" align="center">
+        <Heading component="div" level="1">
+          <IconEnlarge />
+        </Heading>
+      </Stack>,
+    ),
+  alternatives: [],
+  additional: [
+    {
+      label: 'Toggling active state',
+      description: (
+        <Text>
+          Often used to toggle between englarged and reduced states, setting the{' '}
+          <Strong>active</Strong> prop to <Strong>true</Strong> will reverse the
+          icons intent.
+        </Text>
+      ),
+      Example: ({ getState, toggleState }) =>
+        source(
+          <Stack space="large" dividers>
+            <IconEnlarge active={getState('active')} />
+            <Inline space="small">
+              <Button onClick={() => toggleState('active')}>
+                {getState('active') ? 'inactive' : 'active'}
+              </Button>
+            </Inline>
+          </Stack>,
```

Comment:
Do you think an example using a `ButtonIcon` be better here? It's likely more reflective of a real-world use case compared to this which is a bit too trivial IMO.

Reply:
Probably a good point, I followed the existing state toggle examples from the other icon docs (e.g. Bookmark, Heart, etc).

Let me look at updating the set of them

Reply:
Updated all 3 now.